<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsItemClass</class>
 <widget class="QMainWindow" name="QtWidgetsItemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsItem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_tray">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>系统托盘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_jointImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>图片拼接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>图片拼接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_joinUnion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>欢迎控件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>101</width>
      <height>231</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;p style='line-height:200px'&gt;&lt;font style = 'line-height: 400px;font-size:14px; font-weight:bold'&gt;啦啦啦 &lt;/font&gt; &lt;font style = 'color:#2C5DFF; font-size:14px; font-weight:bold'&gt; 通过 &lt;/font&gt; &lt;font style = 'font-size:14px; font-weight:bold'&gt;嘻嘻嘻.&lt;/font&gt;
&lt;/p&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsItem.qrc"/>
 </resources>
 <connections/>
</ui>
